src/shared_utilities/drivers.jl

```diff
@@ -1030,16 +1034,7 @@ function make_update_drivers(r::PrescribedRadiativeFluxes{FT}) where {FT}
         # If we obtain this factor from ERA5 in the future, we should remove thermo_params from the struct.
```

Comment:
I'll clean up this comment. Unfortunately, we dont have diffuse fraction from flux tower sites, so we need to keep the empirical option